packager/media/formats/mp4/segmenter.h

```diff
@@ -147,7 +149,8 @@ class Segmenter {
   ProgressListener* progress_listener_ = nullptr;
   uint64_t progress_target_ = 0u;
   uint64_t accumulated_progress_ = 0u;
-  int32_t sample_duration_ = 0;
+  int32_t sample_durations_[2] = {0, 0};
```

Comment:
These are set from `sample.duration()`, which is `int64_t`.

Also, can you please add this same initializer to to sample_durations_ in ts_muxer.h?